docs/system/UTILS.stories.mdx

```diff
@@ -90,3 +91,191 @@ const calculatedColor = calculateColorBetweenColorAndType(color, type);
 
 console.log(calculatedColor.color, calculatedColor.shade);
 ```
+
+---
+
+### getAAColorFromSwatches
+
+This util is provided from core `theme` or you can use it as a standalone function.
+In case of standalone you must provide the [Palette](/?path=/story/system-theme--page#theme-palette).
```

Comment:
link doesnt work, it should be `[Palette](/story/system-theme--page#theme-palette)`